### PR TITLE
changing anisotropic filter settings

### PR DESCRIPTION
**Pull request by roshanch**
if the user input for anisotropic is greater than supported value by driver, take the maximum supported value.

GearVRf-DCO-1.0-Signed-off-by: Roshan Chaudhari roshan.c@samsung.com

### CODE REVIEW

**Comment by thomasflynn** [comment]
commit message needs DCO and some sort of description ;-)

File: GVRf/Framework/framework/src/main/jni/gl/gl_texture.h [hunk 1]
```diff
@@ -103,10 +103,15 @@ class GLTexture : public GLPendingTask {
             glGenTextures(1, &id_);
             glBindTexture(target_, id_);
 
+            GLfloat anisotropic_level;
+
+            glGetFloatv(GL_MAX_TEXTURE_MAX_ANISOTROPY_EXT, &anisotropic_level);
```
Comment (liaxim) [inline GVRf/Framework/framework/src/main/jni/gl/gl_texture.h]: I believe before making this call you need to be certain EXT_texture_filter_anisotropic is supported.

**Comment by liaxim** [comment]
@roshanch Do you agree the presence of EXT_texture_filter_anisotropic needs to be checked or should we merge as-is?

**Comment by roshanch** [comment]
@liaxim yes, I will add the check for that.

**Comment by roshanch** [comment]
@liaxim added extension check

**Comment by roshanch** [comment]
uhh.!! too many commits from the merge.

**Pull request merged**